PxAPI-2.yml

```diff
@@ -280,8 +280,27 @@ paths:
         '404':
           $ref: '#/components/responses/ErrorResponse404'            
         '429':
-          $ref: '#/components/responses/ErrorResponse429'                
+          $ref: '#/components/responses/ErrorResponse429'
 
+  /config:
+    get:
+      tags:
+        - Administration
+      summary: List all codelists that are associated with the table 

```

Comment:
summary: Return the configuration for this api.

Reply:
Yes, but we should also declare the response schema.